root@ip-172-31-22-87:/home/ubuntu/DevOps_Microservices/project-ml-microservice-kubernetes# ./run_kubernetes.sh
Error from server (AlreadyExists): pods "ml-service" already exists
NAME         READY   STATUS    RESTARTS   AGE
ml-service   1/1     Running   0          70s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000

root@ip-172-31-22-87:/home/ubuntu/DevOps_Microservices/project-ml-microservice-kubernetes# kubectl logs ml-service
* Serving Flask app "app" (lazy loading)
* Environment: production
  WARNING: Do not use the development server in a production environment.
  Use a production WSGI server instead.
* Debug mode: on
* Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
* Restarting with stat
* Debugger is active!
* Debugger PIN: 741-350-132
[2020-06-30 13:42:45,120] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-06-30 13:42:45,133] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-06-30 13:42:45,143] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-06-30 13:42:45,147] INFO in app: Output prediction: [20.35373177134412]
127.0.0.1 - - [30/Jun/2020 13:42:45] "POST /predict HTTP/1.1" 200 -
